### DISCUSSION TITLE
Open VPN android 10 version

### DISCUSSION THREAD

**Discussion by abhi280792** [schwabe/ics-openvpn#1321] [Q&A]
Hello,

I have updated my target sdk version 28 to 29 but and i have paid servers for vpn but its showing below error

E/OpenVPN: OpenVPNThread Got java.lang.NullPointerException: Attempt to invoke virtual method 'void java.lang.Process.destroy()' on a null object reference

Please help me

**Comment by schwabe** [comment]
The app already targets API version 30.

**Reply by abhi280792** [reply]
I know but when i download your code and open on android studio and after successful build, run and showing this error
CreateProcess error=2, The system cannot find the file specified

**Reply by abhi280792** [reply]
Please help me in my code

**Reply by schwabe** [reply]
I don't know what is wrong with your code. But the problem you are describing does not exist in this app.

**Reply by zander-mazda** [reply]
Out of the topic sir schwabe, hope u could help me.

I'm trying to modify a vpn app using apk tool/editor. My source of knowledge about editing is from youtube therefore i am not good at or just a little bit of idea.

My main problem, there are parts that i could not able to edit specific text/string/layout. I dont know where to find that..

Sorry for my poor english. 

Attached is the sample that i wanted to edit.. In an pointed arrow..

Looking forward... Tnx
![AddText_07-08-11 39 47](https://user-images.githubusercontent.com/84930031/124955042-4b919a80-e049-11eb-8f18-d9de0d7cb21b.jpg)
![AddText_07-08-11 48 13](https://user-images.githubusercontent.com/84930031/124955049-4d5b5e00-e049-11eb-90af-dd37342e45a7.jpg)
![AddText_07-08-11 31 21](https://user-images.githubusercontent.com/84930031/124955052-4df3f480-e049-11eb-81eb-379c7d7a8211.jpg)

**Comment by schwabe** [comment]
That is definitively not my app.
